recipes/couchdb/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+set -eux
+
+if [[ $(uname) == Linux ]]; then
+cd $PREFIX
+tar --strip-components=2 -xvf $SRC_DIR/mozjs/data.tar.gz
+tar --strip-components=2 -xvf $SRC_DIR/mozjsdev/data.tar.gz
+chmod +x lib/libmozjs185.so.1.0.0
+patchelf --set-rpath '$ORIGIN' lib/libmozjs185.so.1.0.0
+
+# find .
+# ldd lib/libmozjs185.so
+# readelf -d lib/libmozjs185.so
```

Comment:
This kind of left-overs should be removed or, if you want to keep them for future debugging, add a comment in the line before explaining why this is useful in general.